docs/changes/tests/12633.api.2.rst

```diff
@@ -0,0 +1,3 @@
+Backward-compatible import of ``astropy.tests.helper.remote_data``

```

Comment:
Small grammatical nit-picks (also for the two below)

*The* back-ward ... *has been* removed

Use ... from *the* (or delete "plugin")

Reply:
Is there a dash between "back" and "ward"?

Reply:
Oops, no, there should be no dash there...